### ISSUE TITLE
Forms V8: Autocomplete attributes for form fields

### ISSUE THREAD

**Issue by zagpaulandrews** [umbraco/Umbraco.Forms.Issues#322]
In order to meet WCAG 2.1 AA accessibility guidelines we need the ability to add the "autocomplete" attribute to form fields. Here is a link to the guidelines: https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html

A simple optional text field to enter this value on each form element would be sufficient.

---
_This item has been added to our backlog [AB#12728](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/12728)_

**Comment by nul800sebastiaan** [comment]
Thanks @zagpaulandrews - we're currently not working on new features for a while but this would be a  good one to add to the list when we start working on new features. 

At the moment you will probably need to play a bit with themes -https://our.umbraco.com/documentation/add-ons/umbracoforms/developer/Themes/ and custom fieldtypes, which can have any additional configuration value you need - https://our.umbraco.com/documentation/add-ons/umbracoforms/developer/Extending/Adding-a-Fieldtype.

**Comment by Matthew-Wise** [comment]
Just had a question around this on the Umbraco Slack, be great to see this be in the product instead of something that needs to be developed

**Comment by warrenbuckley** [comment]
Fixed in this PR https://github.com/umbraco/Forms/pull/535 and due for release in 8.8.0
